treadmill-scripts/configureldap.sh

```diff
@@ -116,9 +116,11 @@ trap cleanup EXIT
 cat << E%O%F > ${tmpdir}/LDAP.yaml
 ---
 realm: ${REALM}
+treadmill_cell: None
 treadmill_dns_domain: ${TREADMILL_DNS_DOMAIN}
 treadmill_ldap: ${TREADMILL_LDAP}
 treadmill_ldap_suffix: ${TREADMILL_LDAP_SUFFIX}
+treadmill_krb_realm: ${REALM}
```

Comment:
treadmill_krb_realm or realm?

Reply:
Removed duplicate environment variable and updated cloud-init repo to match.